hail/python/hailtop/config/deploy_config.py

```diff
@@ -104,6 +110,25 @@ async def get_healthcheck(request):  # pylint: disable=unused-argument,unused-va
 
         return root_app
 
+    async def addresses(self, service: str) -> List[Tuple[str, int]]:
+        from ..auth import service_auth_headers  # pylint: disable=cyclic-import
+        namespace = self.service_ns(service)
+        headers = service_auth_headers(self, namespace)
+        async with ssl_client_session(
+                raise_for_status=True,
+                timeout=aiohttp.ClientTimeout(total=5),
+                headers=headers) as session:
+            async with await session.get(
```

Comment:
Is `{namespace}/{service}` treated as the key by the cache? It looks like it should just be the name of the service from the code above.

Reply:
good catch, this is a bug, fixed to just the name